.github/workflows/contrib-tests.yml

```
@@ -0,0 +1,23 @@
+name: contrib-tests
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+.*
+  pull_request:
+
+jobs:
+  contrib_tests:
+    runs-on: ubuntu-latest
+    container:
+      image: cimg/go:1.16
+    steps:
+      - name: Setup Permissions
+        run: sudo chmod -R 777 $GITHUB_WORKSPACE /github /__w/_temp
+      - name: Checkout Repo
+        uses: actions/checkout@v2
```

```suggestion
runs-on: ubuntu-latest
    steps:
      - name: Checkout Repo
        uses: actions/checkout@v2
      - name: Setup Go
        uses: actions/setup-go@v2
        with:
          go-version: 1.17
```
I think we need to run w/ Go 1.17. Likely this change would work, as we use this setup in other actions too.